src/devices/MettlerToledo/Exceptions/MettlerToledoException.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Iot.Device.MettlerToledo.Exceptions
+{
+    /// <summary>
+    /// A generic Mettler Toledo exception
+    /// </summary>
+    public abstract class MettlerToledoException : System.Exception
```

Comment:
I don't feel it's really necessary to have specific exceptions for specific bindings. A simple IOException with the command as a parameter is doing the exact same effect.
IOException are widely used in the repo when there is an issue communicating with the devices.
I've been thru the all code and I'm really not sure that it brings a lot of value to the developer. The difference between the fact a command has been executed badly or create an issue is just that you can't get a proper result. To make it properly, it requires to handle different type of exceptions. And at the end, a simple retry mechanism should be enough.